src/components/TxConfirmDialog/index.js

```diff
@@ -104,16 +97,9 @@ export default class TxConfirmDialog extends Component {
     );
   }
 
-  onClose = () => {
-    this.props.clearTxConfirm();
-  }
-
   onOkClicked = () => {
-    const callback = this.props.txConfirmInfoAndCallback.confirmCallback;
-    if (callback) {
-      callback();
-    }
-    this.props.clearTxConfirm();
+    const { txConfirmInfoAndCallback: { confirmCallback }, clearTxConfirm } = this.props;
+    if (confirmCallback) clearTxConfirm();
```

Comment:
i believe we still need to call `confirmCallback` as the function as well?

Reply:
good catch

Reply:
fixed